### ISSUE TITLE
[FE] 10-10EZ - Household: Remove feature toggle from code

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#62516]
## Background
**NOTE: Do not do this until after the end of November 2023**
Now that we have released the feature at 100% in production, and monitored activity to confirm there are no issues occurring as a result of these changes, we can now remove the feature toggle from the code.

## Tasks
- [x] Remove feature toggle conditionals from vets-website
- [x] Remove feature toggle from vets-API feature list

## Acceptance Criteria
- Feature toggle is removed, as it is no longer needed.

**Comment by jadon1979** [comment]
This is blocked by https://app.zenhub.com/workspaces/10-10-health-apps-5fff0cfd1462b6000e320fc7/issues/gh/department-of-veterans-affairs/va.gov-team/62517 and cannot go up until the migrations have been deployed to prod.

**Comment by jadon1979** [comment]
vets-website https://github.com/department-of-veterans-affairs/vets-website/pull/27677
vets-api https://github.com/department-of-veterans-affairs/vets-api/pull/15296
